### ISSUE TITLE
Parity on video capture with webrtc-uwp-sdk

### ISSUE THREAD

**Issue by aboba** [microsoft/winrtc#21]
The MixedReality SDK is blocked on use of winrtc, as noted [here](https://github.com/microsoft/MixedReality-WebRTC/issues/305).

**Comment by jehumb-havok** [comment]
This is already tracked internally by @LoadLibrary. But I guess it doesn't hurt to have a public-facing issue too. :)

**Comment by LoadLibrary** [comment]
@djee-ms No it doesn't. :)
I'm going to be working on that after //Build.

**Comment by LoadLibrary** [comment]
@aboba When did Insertable Streams landed in Chrome? m83?

**Comment by aboba** [comment]
Looks like it showed up in M82:
https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/Oy84pXDhajI

**Comment by LoadLibrary** [comment]
The parity with WebRTC-UWP is been tracked by #37 , #23 and #35 .

**Comment by fiban-havok** [comment]
AFAICS there's also #12 (should be fixed already, right?) and #32. Also @LoadLibrary you mentioned a regression in #29 that breaks video on Win32, right? Is that confirmed? Should I open another issue for it?
